libs/ui-toolkit/src/components/button/button.tsx

```diff
@@ -18,6 +18,9 @@ const md = 'px-10 py-2 text-base';
 const lg = 'px-14 py-4';
 const fillClasses = 'block w-full';
 const defaultClasses = [
+  'px-10 py-2',
```

Comment:
chore (blocking): defaultClasses is for a button variant (IE border,bg,text colors only) and so shouldn't contain padding/sizing stuff. Additionally this will apply multiple padding classes from which I'm not sure which styles will take precedence.

Suggest moving text-base and text center to the `base` variable. Also it looks like AnchorButton uses size of `lg` by default whereas the regular button doesn't, this is why the sizes are different. Lets just update AnchorButton to use `md` button by default (and remove padding classes) though you'll need to find all usage of AnchorButton and make sure the correct size is used